Apromore-Core-Components/Apromore-Portal/src/main/webapp/themes/ap/common/js/index.js

```diff
@@ -113,4 +113,21 @@ Ap.common.pullClientTimeZone = function () {
  */
 Ap.common.injectGlobalClass = function (klass) {
     jQuery('body')[0].classList.add(klass);
-}
\ No newline at end of file
+}
+
+/**
+ * Encapsulate mailto for issue reporting
+ *
+ * @param email {String} Email
+ * @param username {String} Username
+ */
+Ap.common.reportIssue = function (email, username) {
+    let subject = 'Apromore Issue Report';
+    let ua = (typeof navigator === 'undefined') ? '' : navigator.userAgent.toLowerCase();
+    let href = window.location.href;
+    let body = `Username: ${username}%0D%0A` + `Browser info: ${ua}%0D%0A` + `URL: ${href}%0D%0A`;
+    body += 'Description of your issue:'
+    let url = "mailto:" + email + "?subject=" + subject + "&body=" + body;
```

Comment:
This generates a URL which is sure to contain spaces because of the format of `body`.  Shouldn't it be URL encoded?